packages/utils/classes.ts

```diff
@@ -0,0 +1,3 @@
+export function classes(...cls: string[]): string {
+  return cls.filter(Boolean).join(" ");
```

Comment:
I suggest adding tests for the `classes` function

Reply:
Alright, I agree that it would be ideal, but could you explain how we perform the tests for the project?

Also, how do we run them? Because when checking our list of scripts, I didn't find a `'bun run tests'` and when trying `'bun test'` the following error occurred:

![image](https://github.com/user-attachments/assets/3a0f591b-8a3f-4fd0-b9ae-beae78bf802c)

Reply:
Bun test is not configured, this test is the integration test (e2e tests). Yo run it, you need to run `bun run e2e`.

This function should be **unit-tested**. For this, we create a sibling file with the same name + `.test` at the end. So, in this case `packages/utils/classes.test.ts`

1. first you need to configure the runtime to only test `.test.ts` files, and ignore `.spec.ts`
2. add the test runner script to the `package.json` scripts (if needed)
3. now you can create the tests
4. run the tests
5. (extra) run the unit-tests on each PR (quality-gateway for github action)


- https://bun.sh/docs/cli/test

Reply:
What I did:

1. I couldn't find in the `bun` documentation how to change the 'glob' filters, so I added a filter to make the tests only search for files in the `packages` folder
2. Added the script in the package.json
   - This was necessary to configure the filter mentioned in point 2
3. Created tests for the 'classes' utils
4. Tests ran successfully and passed 100%
   ![image](https://github.com/user-attachments/assets/ef49eb2f-5405-4c3e-9969-4b259287343b)
5. I didn't change the action or the lefthook because they are using `yarn`, and the tests only work with `bun`

Reply:
Awesome! Great job!

Regarding item 5, I'll raise a discussion on our discord to choose if we'll keep yarn or use bun